### ISSUE TITLE
in CreateTun - call to luid.SetRoutes does not complete successfully

### ISSUE THREAD

**Issue by dovholuknf** [openziti/desktop-edge-win#111]
User is running Win10 running in Virtualbox. Installed the 1.0.5 installer. On upgrade using the installer [the error reported in 110](https://github.com/openziti/desktop-edge-win/issues/110) was displayed. I asked for logs from the service and saw an error I'd not seen before:

```
[34m[  24.155][39m [37m   INFO[39m [36m[39m: routing destination [{169.254.1.0 ffffff00}] through [169.254.1.0]
[34m[  24.155][39m [31m  ERROR[39m [36m[39m: unexpected err: Element not found.
[34m[  24.155][39m [31m  ERROR[39m [36m[39m: the main loop exited with an unexpected error: Element not found.
[34m[  24.155][39m [37m   INFO[39m [36m[39m: waiting for shutdown to complete
```

At commit `bd937dbf5bc01788e3be089d6f11ba770ebe78c7` this was tracked back (using other log messages) to be called from `func (t *RuntimeState) CreateTun(ipv4 string, ipv4mask int) error {` 
![image](https://user-images.githubusercontent.com/46322585/90516599-3b6f8480-e132-11ea-83b8-3a346fda7a59.png)

you can see from the code shown above and the log messages this line was not in the logs: `log.Info("routing applied")`

If we can discern what went wrong with the wintun code (go pkg:`"golang.zx2c4.com/wireguard/windows/tunnel/winipcfg"`) we should look to contribute back to wintun

**Comment by dovholuknf** [comment]
this seems to be a race condition that has not been tracked down yet. putting breakpoints in this code and stepping through can trigger this exact error. Also - issue #119 had a similar "Element not found" error due to IPv6 being totally disabled

**Comment by dovholuknf** [comment]
fixed in 1.4.x timeframe. this was a timing issue. the TUN was not available. fixed by adding a retry loop followed by a retry timeout.  10s is allocated for the interface to become available. for 7.5s a debug message will be produced. from 8 to 10 seconds a warning shows in the logs and a panic triggers at 10s
